deltachat-ios/Controller/QrViewController.swift

```diff
@@ -43,6 +43,14 @@ class QrViewController: UIViewController {
         title = String.localized("qrshow_title")
         setupSubviews()
         view.backgroundColor = DcColors.defaultBackgroundColor
+
```

Comment:
because of style reasons and consistency over the project I would initialize these UI elements lazily.

Reply:
hm, yes, maybe, i also thought about that. but - we are not doing that everywhere and, tbh, i do not really get the gist of tearing code artificially that way :)

but regarding overall style and consistency, it is probably better.